integreat_cms/cms/templatetags/pdf_filters.py

```diff
@@ -30,3 +30,30 @@ def pdf_strip_fontstyles(instance):
         if style := element.attrib.pop("style", None):
             element.attrib["style"] = re.sub(r"font-[a-zA-Z]+:[^;]+", "", style)
     return unescape(tostring(content, with_tail=False).decode("utf-8"))
+
+
+@register.filter
+def pdf_truncate_links(instance, max_chars):
+    """
+    This tag returns the instance with truncated links.
+
+    :param instance: The content object instance
+    :type instance: ~integreat_cms.cms.models.pages.page.Page
+
+    :param max_chars: The maximal length of a link before it will be truncated
+    :type max_chars: int
+
+    :return: The instance with truncated links
+    :rtype: ~integreat_cms.cms.models.pages.page.Page
+    """
+    try:
+        content = fromstring(instance)
+    except ParserError:
+        return instance
+    truncation_postfix = "[...]"
+    truncation_length = max_chars - len(truncation_postfix)
+    for elem, _, _, _ in content.iterlinks():
```

Comment:
I'm not very familiar with Python conventions, so this is more of a question:
Isn't it better to use meaningful names, even if these variables are not used?

Reply:
Good point. In general, the underscore is a very common convention in Python for unused variables, and we also use this in our code base. However, it's not completely ideal because it clashes with the alias import for translations, e.g. imagine
```
from django.utils.translation import gettext as _
```
on top of a file, then having the "unused" pattern
```
var, _ = some_func()
```
and later you want to translate something
```
text = _("Translated text")
```
This won't work, since `_` has been overwritten.
So yes, I's agree that in this case it's better to use readable variable names and then ignore the corresponding pylint warning.

Reply:
Okay, I will change that, the translation alias was not on my mind, makes totally sense!